src/pages/MainPage/MainPage.tsx

```diff
@@ -18,10 +19,24 @@ const MainPage = () => {
     };
   }, {});
 
+  const optionsForSort = {
+    releaseDate: 'Release date',
+    rating: 'Rating',
+  };
+
   return (
     <Layout>
       <div className={styles.filterPanel}>
         <FilterByTags options={optionsForFilter} />
+        <Sort
+          caption='Sort by'
+          id='sorting-movies'
+          name='sorting-movies'
+          value=''
+          options={optionsForSort}
+          onChange={() => {}}
+          className={styles.sort}
+        />
       </div>
 
```

Comment:
Additionally, let's simulate the async action of getting the data here.
Under "src/core" folder that you create after one of the above comments consider creating a folder "api" folder that will hold all the specific API requests in your app.
The files under "src/core/api" could be divided into 2 groups: mocked and real.
Use Promise.resolve fro actual async effect and useEffect to get the data.
That's going to be used for "movies.json" primarily.